srcpkgs/cage/patches/cage-0.1.1-fix-building-with-wlroots-0.9.1.patch

```diff
@@ -0,0 +1,77 @@
+--- output.c
++++ output.c
+@@ -386,6 +386,9 @@
+ 	/* Place the cursor in the center of the screen. */
+ 	wlr_cursor_warp(server->seat->cursor, NULL, wlr_output->width / 2, wlr_output->height / 2);
+ 	wlr_output_damage_add_whole(server->output->damage);
++
++	wlr_output_enable(wlr_output, true);
++	wlr_output_commit(wlr_output);
+ }
+ 
+ void
+--- .builds/alpine.yml
++++ .builds/alpine.yml
+@@ -17,8 +17,7 @@
+   # version, instead of master, to avoid any breaking changes in wlroots.
+   - wlroots: |
+       cd wlroots
+-      # This corresponds to the tag of 0.6.0
+-      git checkout c0305f4f864543f8c3fea6f302e91c9b1d3396f3
++      git checkout 0.9.1
+       meson --prefix=/usr build -Drootston=false -Dexamples=false
+       ninja -C build
+       sudo ninja -C build install
+--- .builds/archlinux.yml
++++ .builds/archlinux.yml
+@@ -15,8 +15,7 @@
+   # version, instead of master, to avoid any breaking changes in wlroots.
+   - wlroots: |
+       cd wlroots
+-      # This corresponds to the tag of 0.6.0
+-      git checkout c0305f4f864543f8c3fea6f302e91c9b1d3396f3
++      git checkout 0.9.1
+       meson --prefix=/usr build -Drootston=false -Dexamples=false
+       ninja -C build
+       sudo ninja -C build install
+--- .builds/freebsd.yml
++++ .builds/freebsd.yml
+@@ -18,8 +18,7 @@
+   # version, instead of master, to avoid any breaking changes in wlroots.
+   - wlroots: |
+       cd wlroots
+-      # This corresponds to the tag of 0.6.0
+-      git checkout c0305f4f864543f8c3fea6f302e91c9b1d3396f3
++      git checkout 0.9.1
+       meson --prefix=/usr/local build -Drootston=false -Dexamples=false
+       ninja -C build
+       sudo ninja -C build install
```

Comment:
... to this line. As this is just for the builds they use interally and dosen't impact on our builds.

Reply:
Right, tested it on my side and it seems to working fine, will commit in a while.